workshop/emr-eks/cleanup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+read -p "Enter the region: " region
+export AWS_DEFAULT_REGION=$region
```

Comment:
If they are using the AWS provided account then they don't need to run cleanup scripts as the Accounts will be deleted. 

If they are using their local mac or cloud9 IDE using their own account then they can run `clanup.sh` script from the terminal. Terminal will prompt for the region to avoid issues with incorrect region.